accumulator/pollard_test.go

```diff
@@ -215,6 +226,52 @@ func fixedPollard(leaves int32) error {
 	return nil
 }
 
+func PollardSerializeDeserialize() error {
+	var p Pollard
+
+	// generate slice of leaf
+	leaves := make([]Leaf, 10)
+
+	for i := 0; i < len(leaves); i++ {
+		leaves[i].Hash[0] = uint8(i + 1)
+	}
+
+	// add leaves to pollard
+	p.add(leaves)
+
+	// performing serialization
+
+	old_byte, err := p.Serialize()
+
+	if err != nil {
+		return err
+	}
+
+	// perform Deserialize
+
+	err = p.Deserialize(old_byte)
+	if err != nil {
+		return err
+	}
+
+	// Serialize again and compare bytes
+
+	new_byte, err := p.Serialize()
+
+	if err != nil {
+		return err
+	}
+
+	res := bytes.Compare(old_byte, new_byte)
```

Comment:
There's a `bytes.Equal()` that outputs a bool (true if equal, false if not equal). Would be better to use that instead of `bytes.Compare()`.